src/main/java/pl/wrapper/parking/facade/domain/DummyController.java

```diff
@@ -0,0 +1,36 @@
+package pl.wrapper.parking.facade.domain;
+
+import ch.qos.logback.core.joran.sanity.Pair;
+import jakarta.servlet.http.HttpServletRequest;
+import lombok.RequiredArgsConstructor;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.RestController;
+import pl.wrapper.parking.infrastructure.error.*;
+import pl.wrapper.parking.infrastructure.error.Error;
+
+@RestController
+@RequiredArgsConstructor
+public class DummyController extends HandleResult {
+    private final DummyService dummyService;
+
+    @GetMapping("/id/{id}")
+    public ResponseEntity<String> getParkingOccupancyByParkingIdValid(
+            HttpServletRequest request, @PathVariable("id") Long id) {
```

Comment:
why do we take request as an argument here? We don't need it mapped so remove it